app/models/ingredient.rb

```diff
@@ -1,3 +1,9 @@
 class Ingredient < ApplicationRecord
    validates_presence_of :name, :cost
+   has_many :recipe_ingredients
+   has_many :recipes, through: :recipe_ingredients
+
+   def self.sort_alpha
+      self.order(name: :asc)
+   end
 end
```

Comment:
Great job with your AR. `:asc` is the default so you could probably omit it if you wanted.